modules/network-observability-configuring-predefined-alerts.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// network_observability/network-observability-alerts.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="network-observability-configuring-predefined-alerts_{context}"]
+= Configuring predefined alerts
+////
+:FeatureName: Network observability alerts
+include::snippets/technology-preview.adoc[]
+////
+[role="_abstract"]
+Alerts in the Network Observability Operator are defined using alert templates and variants in the `spec.processor.metrics.alerts` object of the `FlowCollector` custom resource. You can customize the default templates and variants for flexible, fine-grained alerting.
+
+You can configure alerts for the Network Observability Operator in the `FlowCollector` custom resource (CR), and monitor alerts through the *Network Health* dashboard in the *Observ* section of the {product-title} web console. Alerts are organized by the default alert templates and variants. For more information, see "About network observability alerts".
```

Comment:
These two paragraphs tell roughly the same thing, I have the feeling one is the rewriting of the other and you missed deleting the old ? :-)